task/createPullRequest.ps1

```diff
@@ -651,6 +676,85 @@ function SetAutoComplete {
     }
 }
 
+function BypassPR {
```

Comment:
Hi! why did you create a new function and not just update the SetAutoComplete function?

Reply:
Hello, because you need to pass other parameters to Function like "status" and "lastMergeSourceCommit" and also you cannot pass the parameter "autoCompleteSetBy" because it presents the error saying that it does not accept both together (bypass and autocomplete)

Reply:
Oh ok I understand, you can still "play" with it inside the function but it's ok :)